TCMSApp/src/server/routes/login.js

```diff
@@ -0,0 +1,99 @@
+var express = require('express');
+var router = express.Router();
+var passport = require('passport');
+var localStrategy = require('passport-local');
+var User = require('../model/mschemas/user.js');
+var bcrypt = require('bcrypt-nodejs');
+var trelloStrategy = require('passport-trello').Strategy;
+
+var TRELLO_ID = '661d5fc5e58a01d692762e9165414c4f';
+var TRELLO_SECRET = '9a3f476a48bc202d0eaee4773f7aa7fa3b5d60e46c88e10c6289b0bfff7ea7f2';
+var TRELLO_CALLBACK = 'http://localhost:3000/register/trello/callback';
+
+passport.serializeUser(function(user, done) {
+    done(null, user.id);
+});
+
+passport.deserializeUser(function(id, done) {
+    User.findById(id, function(err, user) {
+        done(err, user);
+    });
+});
+
+passport.use('trello', new trelloStrategy({
+        consumerKey: TRELLO_ID,
+        consumerSecret: TRELLO_SECRET,
+        callbackURL: TRELLO_CALLBACK,
+        passReqToCallback: true,
+        trelloParams: {
+            scope: 'read,write',
+            name: 'TCMS',
+            expiration: 'never'
+        }
+    },
+    function(req, token, tokenSecret, profile, done) {
+        if (profile) {
+            var findOrCreateUser = function() {
+                User.findOne({'trelloUserID': profile.id},function(err, user) {
+                    if (user) {
+                        console.log('Everything is ok');
+                        return done(null, user);
+                    } else {
+                        var newUser = new User();
+                        newUser.fullName = profile.displayName;
+                        newUser.role = 'administrator';
+                        newUser.trelloUserID = profile.id;
+
+                        newUser.save(function(err) {
```

Comment:
it seems that you're creating the user in DB here during trello login. But I am entering my local mongo db and don't see any user.
Do you know why?